conformance/reports/v1.2.1/envoy-gateway/experimental-v1.3.0-default-report.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: gateway.networking.k8s.io/v1
+date: "2025-01-31T03:14:06Z"
+gatewayAPIChannel: experimental
+gatewayAPIVersion: v1.2.1
+implementation:
+  contact:
+  - https://github.com/envoyproxy/gateway/blob/main/GOVERNANCE.md
+  organization: envoyproxy
+  project: envoy-gateway
+  url: https://github.com/envoyproxy/gateway
+  version: latest
+kind: ConformanceReport
+mode: default
+profiles:
+- core:
+    result: success
+    statistics:
+      Failed: 0
+      Passed: 11
+      Skipped: 0
+  extended:
+    result: partial
+    skippedTests:
+    - GatewayInfrastructure
+    - GatewayStaticAddresses
+    statistics:
+      Failed: 0
+      Passed: 0
+      Skipped: 2
+    supportedFeatures:
+    - GatewayHTTPListenerIsolation
+    - GatewayInfrastructurePropagation
+    - GatewayPort8080
+    - GatewayStaticAddresses
+  name: GATEWAY-TLS
+  summary: Core tests succeeded. Extended tests partially succeeded with 2 test skips.
+- core:
+    result: success
+    statistics:
+      Failed: 0
+      Passed: 12
+      Skipped: 0
+  extended:
+    result: partial
+    skippedTests:
+    - GatewayInfrastructure
+    - GatewayStaticAddresses
+    statistics:
+      Failed: 0
+      Passed: 0
+      Skipped: 2
+    supportedFeatures:
+    - GatewayHTTPListenerIsolation
+    - GatewayInfrastructurePropagation
+    - GatewayPort8080
+    - GatewayStaticAddresses
+  name: GATEWAY-GRPC
+  summary: Core tests succeeded. Extended tests partially succeeded with 2 test skips.
+- core:
+    result: success
+    statistics:
+      Failed: 0
+      Passed: 33
+      Skipped: 0
+  extended:
+    result: partial
+    skippedTests:
+    - GatewayInfrastructure
+    - GatewayStaticAddresses
```

Comment:
Why skip tests and claim support for features? Why not just drop `GatewayInfrastructurePropagation` as a supported feature for example?

Reply:
its hard to keep track of which features are added in upstream, Envoy Gateway would like to support all features except mesh, which is why we define the suite options this way https://github.com/envoyproxy/gateway/blob/cbd5bc519ed6aa80adcac99a61ec1acf2822ef9c/internal/gatewayapi/conformance/suite.go#L30

Reply:
Really love that goal, but if it's combined with skipped tests it can lead to confusion in our conformance reporting unfortunately. This will make it look like Envoy Gateway supports `GatewayInfrastructurePropagation` while the skipped test suggests it doesn't.

Reply:
It sounds like some sort of reporting of "what features exist but have not been opted-in to" would be helpful here, at least for like local debug output.